gravity.sh

```diff
@@ -43,7 +45,7 @@ else
 fi
 
 #Remove the /* from the end of the IPv4addr.
-IPv4addr=${IPv4_address%/*}
+IPv4_address=${IPv4_address%/*}
```

Comment:
dose dhcpd call this file for this var tho?  all of the scripts in this branch that i have looked at that use this var remove the cidr so if it was stored in setupVars.conf without a cidr less compute used all together